Sources/Helpers/OAUtilities.m

```diff
@@ -332,6 +332,7 @@ - (void) addBlurEffect:(BOOL)light cornerRadius:(CGFloat)cornerRadius padding:(C
                 ? UIBlurEffectStyleLight : UIBlurEffectStyleDark];
 
     UIVisualEffectView *blurView = [[UIVisualEffectView alloc] initWithEffect:blurEffect];
+    blurView.tag = -999;
```

Comment:
method addBlurEffect should check for UIAccessibilityIsReduceTransparencyEnabled() (see usage in the project)